benchmarks/results/s3.md

```diff
@@ -0,0 +1,13 @@
+
+## AWS S3 Benchmark
+
+### 2020/10
+
+```
+Benchmark                                          Mode  Cnt   Score    Error  Units
+S3DownloadBenchmark.akkaDownload                  thrpt   10  77.798 ± 10.162  ops/s
+S3DownloadBenchmark.monixDownload                 thrpt   10  46.995 ±  4.509  ops/s
+S3DownloadBenchmark.monixDownloadMultipart        thrpt   10  37.617 ±  5.914  ops/s
+S3DownloadBenchmark.monixUnsafeDownload           thrpt   10  44.193 ±  4.781  ops/s
+S3DownloadBenchmark.monixUnsafeDownloadMultipart  thrpt   10  40.349 ±  4.191  ops/s
+```
```

Comment:
@Avasil @alexandru it seems like the _akka s3_ integration outperforms the _monix_ one, I believe that the principal reason is because akka uses http request directly instead of using the _aws java sdk_. I guess that if we would like to reach that point we would have to do the same and completely change the current impl. (Just letting you know)